### ISSUE TITLE
Training with negative samples

### ISSUE THREAD

**Issue by arieltanbear** [OlafenwaMoses/ImageAI#486]
Hi Moses,
     First of all, this is a great work that gives us such a convenient way to train and predict.
     I want to add some images without any labelled boxes as negative samples to my dataset. What I did was flowed the method given here:["qqwweee/keras-yolo3#294" ](https://github.com/qqwweee/keras-yolo3/issues/294)
    "make annotations without coordinates or labels"
    But this seems does not work in ImageAI. I also noticed that the line 60 in voc.py:
` 
        if len(img['object']) > 0:
                all_insts += [img]
`
    This will let annotations only with coordinates or labels to be as training samples. so I tried to change it to the code:
`
            if len(img['object']) == 0:
                img['object'] = {}
`
`
            all_insts += [img]
`
     Force to add these negative samples to training samples. After training, I tested the model with some samples just in my training set. I found that it seemed predict all right with my positive samples. But there are hundreds and thousands of objects are predicted with high probability(like 80%,90%) with my negative samples. I do not know what happened. These negative samples are expected to predict nothing. 
       Is there any method to solve this?

**Comment by alexvmt** [comment]
@arieltanbear I'm currently thinking about the same issue. In my training set I have images without objects that I would like to use for training because in my test set there are also images without any objects. Did you figure out how to do it?
